webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/NodeFactQueryProcessor.scala

```diff
@@ -196,8 +196,7 @@ class NodeFactQueryProcessor(
     status:        InventoryStatus = AcceptedInventory
 ) extends QueryProcessor with QueryChecker {
 
-  def process(query:       Query)(implicit qc: QueryContext): Box[Seq[NodeId]] = processPure(query).map(_.toList.map(_.id)).toBox
-  def processOnlyId(query: Query)(implicit qc: QueryContext): Box[Seq[NodeId]] = processPure(query).map(_.toList.map(_.id)).toBox
+  def process(query: Query)(implicit qc: QueryContext): Box[Seq[NodeId]] = processPure(query).map(_.toList.map(_.id)).toBox
```

Comment:
`processOnlyId` is in fact actually exactly the same as `process`, so it just add useless noise